src/libs/MoneyRequestUtils.ts

```diff
@@ -40,10 +40,10 @@ function addLeadingZero(amount: string): string {
 /**
  * Calculate the length of the amount with leading zeroes
  */
-function calculateAmountLength(amount: string, decimals: number): number {
```

Comment:
@bernhardoj Got a type error here since you removed `decimals` but still passing 2 arguments down in the `validateAmount` function return.

Reply:
Fixed